persistence/selecting.md

```diff
@@ -29,18 +29,15 @@ graph TD
 start((start)) --> A{Using sagas<br /> or outbox?}
 A --> |Yes|B{Additional<br />requirements?}
 A --> |No| I{Which transport?}
-I --> |Other|L[SQL]
-I --> |RabbitMQ/Azure Service Bus|J[No persistence]
-I --> |Azure Storage Queues|K[Azure Table]
+I --> |MSMQ|L[SQL]
+I --> |All other transports|J[None required]
+B --> |MongoDB|M[MongoDB]
 B --> |Service Fabric|C[Service Fabric]
 B --> |RavenDB|E[RavenDB]
-B --> |MongoDB|M[MongoDB]
 B --> |Azure|Q{IaaS/PaaS ?}
```

Comment:
Small thing, but this looks slightly wonky:

![image](https://user-images.githubusercontent.com/677704/128006382-3f36e1c7-d1d9-4b0e-8980-0490b19756ff.png)

```suggestion
B --> |Azure|Q{PaaS/IaaS ?}
```